project.clj

```diff
@@ -13,4 +13,4 @@
    :1.6 {:dependencies [[org.clojure/clojure "1.6.0"]]}
    :1.7 {:dependencies [[org.clojure/clojure "1.7.0"]]}
    :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]]}
-   :1.9 {:dependencies [[org.clojure/clojure "1.9.0-alpha11"]]}})
+   :1.9 {:dependencies [[org.clojure/clojure "1.9.0"]]}})
```

Comment:
Remove change - it's not directly relevant to the PR so we can handle this in a separate commit.